src/common/dcsvg.cpp

```diff
@@ -490,6 +676,48 @@ void wxSVGFileDCImpl::DoDrawPolygon(int n, const wxPoint points[],
     write(s);
 }
 
+void wxSVGFileDCImpl::DoDrawPolyPolygon(int n, const int count[], const wxPoint points[],
+                                  wxCoord xoffset, wxCoord yoffset,
+                                  wxPolygonFillMode fillStyle)
+{
+    if (n == 1)
+    {
+        DoDrawPolygon(count[0], points, xoffset, yoffset, fillStyle);
+        return;
+    }
+
+    int i, j;
+    int totalPts = 0;
+    for (j = 0; j < n; ++j) {
+        totalPts += count[j];
+    }
+
+    wxPoint* pts;
+    pts = new wxPoint[totalPts + n];
```

Comment:
Could we please use `wxScopedArray<wxPoint> pts(totalPts + n)` instead for the peace of mind?